$OpenBSD$

Teach hashable-1.4.3.0 to declare os-string when building with GHC 9.10.

--- hashable-1.4.3.0/hashable.cabal
+++ hashable-1.4.3.0/hashable.cabal
@@ -89,6 +89,9 @@
   if !impl(ghc >=9.2)
     build-depends: base-orphans >=0.8.6 && <0.10
 
+  if impl(ghc >=9.10)
+    build-depends: os-string >=2.0.7 && <2.1
+
   if !impl(ghc >=9.4)
     build-depends: data-array-byte >=0.1.0.1 && <0.2
